draft-ietf-masque-h3-datagram.md

```diff
@@ -198,40 +114,14 @@ so the largest legal value of Quarter Stream ID is (2<sup>62</sup>-1) / 4.
 Receipt of a frame that includes a larger value MUST be treated as a connection
```

Comment:
Noticed in reviewing the whole...  The line above says `(2<sup>62</sup>-1) / 4` where `2<sup>60</sup>-1` would do.

Reply:
Agreed, made the change.